test/integration/inventory/metadata.js

```diff
@@ -80,6 +81,15 @@ describe('(/inventory/metadata) The inventory metadata http API', () => {
       .catch(helpers.handler);
   });
 
+  it('GET /inventory/metadata?sellable=1 returns the list of sellable inventories', () => {
+    return agent.get('/inventory/metadata?sellable=1')
+      .then(res => {
+        expect(res).to.have.status(200);
+        expect(res.body).to.not.be.empty;
```

Comment:
Hmm... this test only checks that the route doesn't break.   Let's make sure the logic is correct.

How about you change one of the default inventory items to `sellable = 0` and then check that `agent.get('/inventory/metadata?sellable=1')` does not include that item, but `agent.get('/inventory/metadata')` does?